src/services/metadata.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { PredefinedProperties } from '../utils/types';
+import { fetchAppsMetadata, MetadataJson } from './apps-metadata';
+
+export interface Metadata {
+    name: string;
+    url: string;
+    appColor: string;
+    hiddenInAppsMenu: boolean;
+    resources: unknown;
+}
+
+export interface StudyMetadata extends Metadata {
+    name: 'Study';
+    predefinedEquipmentProperties: {
+        [networkElementType: string]: PredefinedProperties;
+    };
```

Comment:
@TheMaskedTurtle a preference?
```suggestion
    predefinedEquipmentProperties: Record<string, PredefinedProperties>;
```

Reply:
Is it exactly the same ?
If so, I prefer the first one, because the name can help to understand the object.

Reply:
yes the record is better

Reply:
I mean the other one x) don't you think networkElementType help to understand the object ?